docs/tutorials/docker-compose.yml

```diff
@@ -41,7 +41,7 @@ services:
       SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: zookeeper:32181
 
   ksql-server:
-    image: "confluentinc/ksqldb-server:latest"
+    image: "confluentinc/cp-ksqldb-server:latest"
```

Comment:
This would pull the Confluent Platform image, rather than the Standalone image, i.e. 5.5.0 rather than 0.9.0. Is this intentional?